### PR TITLE
Fix EZP-25292: fix stale relations for non-translatable attributes.

### PR DESCRIPTION
**Pull request by joaoinacio**
JIRA: https://jira.ez.no/browse/EZP-25292

### CODE REVIEW

**Comment by glye** [comment]
+1 afaict, if it affects only object relation and not object relation list.

**Comment by yannickroger** [comment]
+1

**Comment by joaoinacio** [comment]
@glye: thanks, It didn't look like it but I am double-checking about relationlist before merging.

**Comment by joaoinacio** [comment]
script updated for relationlist (for now, in a separate commit for readability).
re ping @glye @yannickroger

File: update/common/scripts/5.4/cleanuntranslatablerelations.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+$script->initialize();
+$db = eZDB::instance();
+
+if ( $optDryRun ) {
+    $cli->warning( "dry-run mode" );
+}
+
+
+/**
+ * Parse xml_text from ezobjectrelatinolist attribute and return array of related content ids
```
Comment (yannickroger) [inline update/common/scripts/5.4/cleanuntranslatablerelations.php]: typo ezobjectrelatinolist

**Comment by yannickroger** [comment]
still +1

**Comment by glye** [comment]
looks good, still +1

**Pull request merged**